tasks/shared_libraries/recovery/shared_recovery.py

```diff
@@ -5,86 +5,106 @@
 from enum import Enum
 import json
 import boto3
-from typing import Dict,Any, Optional
+from typing import Dict, Any, Optional
+
+# sqs = boto3.client("sqs")
+# internal_sqs = boto3.client("sqs")
 
-sqs = boto3.client('sqs')
 
 class RequestMethod(Enum):
     """
     Provides potential actions for the database lambda to take when posting to the SQS queue.
     """
-    NEW = 'post'
-    UPDATE = 'put'
+
+    NEW = "post"
+    UPDATE = "put"
+
 
 class OrcaStatus(Enum):
     """
     Defines the status value used in the ORCA Recovery database for use by the recovery functions.
     """
+
     PENDING = 1
-    STAGED  = 2
+    STAGED = 2
     SUCCESS = 3
-    FAILED  = 4
-
-def post_status_for_job_to_queue(job_id: str, granule_id: str, status_id: OrcaStatus,
-                                 request_time: Optional[str], completion_time: Optional[str],
-                                 archive_destination: Optional[str],
-                                 request_method: RequestMethod, db_queue_url: str):
-    
-    new_data = {'job_id': job_id, 'granule_id': granule_id}
+    FAILED = 4
+
+
+def post_status_for_job_to_queue(
+    job_id: str,
+    granule_id: str,
+    status_id: OrcaStatus,
+    request_time: Optional[str],
+    completion_time: Optional[str],
+    archive_destination: Optional[str],
+    request_method: RequestMethod,
+    db_queue_url: str,
+):
+
```

Comment:
Add doc string. See previous comments on other items to cleanup in the shared library.

Reply:
@AndrewDorn can you provide the doc string?